src/renderer/components/+workloads-pods/__tests__/pod-details-list.test.tsx

```diff
@@ -0,0 +1,288 @@
+
+const runningPod = new Pod({
+  apiVersion: "foo",
+  kind: "Pod",
+  metadata: {
+    name: "foobar",
+    resourceVersion: "foobar",
+    uid: "foobar",
+    ownerReferences: [{
+      uid: "runningReplicaSet",
+      apiVersion: "v1",
+      kind: "ReplicaSet",
+      name: "running",
+    }],
+    namespace: "default",
+    selfLink: "/apis/apps/v1/replicasets/default/foobar",
+  },
+  status: {
+    phase: "Running",
+    conditions: [
+      {
+        type: "Initialized",
+        status: "True",
+        lastProbeTime: 1,
+        lastTransitionTime: "1",
+      },
+      {
+        type: "Ready",
+        status: "True",
+        lastProbeTime: 1,
+        lastTransitionTime: "1",
+      },
+    ],
+    hostIP: "10.0.0.1",
+    podIP: "10.0.0.1",
+    startTime: "now",
+    containerStatuses: [],
+    initContainerStatuses: [],
+  },
+});
+
+const pendingPod = new Pod({
+  apiVersion: "foo",
+  kind: "Pod",
+  metadata: {
+    name: "foobar-pending",
+    resourceVersion: "foobar",
+    uid: "foobar-pending",
+    ownerReferences: [{
+      uid: "pendingReplicaSet",
+      apiVersion: "v1",
+      kind: "ReplicaSet",
+      name: "pending",
+    }],
+    namespace: "default",
+    selfLink: "/apis/apps/v1/replicasets/default/foobar-pending",
+  },
+});
+
+const failedPod = new Pod({
+  apiVersion: "foo",
+  kind: "Pod",
+  metadata: {
+    name: "foobar-failed",
+    resourceVersion: "foobar",
+    uid: "foobar-failed",
+    ownerReferences: [{
+      uid: "failedReplicaSet",
+      apiVersion: "v1",
+      kind: "ReplicaSet",
+      name: "failed",
+    }],
+    namespace: "kube-system",
+    selfLink: "/apis/apps/v1/replicasets/default/foobar-failed",
+  },
+  status: {
+    phase: "Failed",
+    conditions: [],
+    hostIP: "10.0.0.1",
+    podIP: "10.0.0.1",
+    startTime: "now",
+  },
+});
+
+describe("<PodDetailsList />", () => {
+  let di: DiContainer;
+  let podStore: PodStore;
+  let render: DiRender;
+  let fetchMock: AsyncFnMock<Fetch>;
+
+  beforeEach(() => {
+    di = getDiForUnitTesting({ doGeneralOverrides: true });
+    di.override(storesAndApisCanBeCreatedInjectable, () => true);
+    di.override(
+      directoryForLensLocalStorageInjectable,
+      () => "/some-directory-for-lens-local-storage",
+    );
+
+    fetchMock = asyncFn();
+    podStore = di.inject(podStoreInjectable);
+
+    di.override(fetchInjectable, () => fetchMock);
+
+    render = renderFor(di);
+  });
+  
+  describe("when no pods passed and podStore is loaded", () => {
+    beforeEach(() => {
+      podStore.isLoaded = true;
+    });
+
+    it("renders", () => {
+      const result = render(
+        <PodDetailsList
+          pods={[]}
+        />,
+      );
+
+      expect(result.container).toMatchSnapshot();
+    });
+
+    it("shows empty message", () => {
+      const result = render(
+        <PodDetailsList
+          pods={[]}
+        />,
+      );
+
+      expect(result.getByText("No items found")).toBeInTheDocument();
+    });
+  });
```

Comment:
Since all these test doesn't related to the PR's topic and are common random checks for UI component we're decided to move to separated PR _(maybe with some additional/more precise tests)_